spec/template.dd

```diff
@@ -1082,6 +1082,21 @@ $(H2 $(LNAME2 function-templates, Function Templates))
         }
         ---
 
+    $(P Variadic Function Templates can have parameters with default values.
+        These parameters are always set to their default value in case of IFTI.
+    )
+
+        $(SPEC_RUNNABLE_EXAMPLE_COMPILE
+        ---
+        void fun(T...)(T t, string file = __FILE__)
+        {
+            import std.stdio;
+            writeln(file, " ", t);
+        }
+        fun(1, "foo");  // T.length = 2, not 1 (IFTI)
+        fun!int(1, "foo");  // T.length = 1
+        ---
```

Comment:
you need to close the ddoc macro here.

Reply:
done

Reply:
this always print `1` for me:

![image](https://user-images.githubusercontent.com/4370550/36057787-8edb2574-0e13-11e8-9bf6-afd4b64ab1af.png)

https://run.dlang.io/is/CKBf6g

Reply:
works from git master

Reply:
No it uses the the DMD nightlies and updates itself daily.
Though I think that's the root of the problem:

```
~/dlang/install.sh dmd-nightly
Downloading and unpacking http://nightlies.dlang.org/dmd-master-2018-02-01/dmd.master.linux.tar.xz
```

And yeah - there's a build error:  http://nightlies.dlang.org/dmd-master-2018-02-10/build.html
Why is there no loud bot in place that complains about this? :/

Reply:
Anyhow, maybe we should use `_RUN` + `assert` here? Then the spec tester will work like a testsuite.

Reply:
done